src/main/resources/com/google/api/codegen/php/standalone_sample.snip

```diff
@@ -326,3 +328,77 @@
         $unhandledCase: {@initValue.type}$
     @end
 @end
+
+@private processResponse(sample)
+    #{@sample.outputs}
```

Comment:
Is this line supposed to be in the output? Won't the `#` character make it into a comment in the snippet?